web/src/main/java/no/nav/ung/sak/web/server/abac/PdpRequestMapper.java

```diff
@@ -0,0 +1,72 @@
+package no.nav.ung.sak.web.server.abac;
+
+import no.nav.k9.felles.sikkerhet.abac.PdpRequest;
+import no.nav.sif.abac.kontrakt.abac.*;
+import no.nav.sif.abac.kontrakt.abac.dto.OperasjonDto;
+import no.nav.sif.abac.kontrakt.abac.dto.SaksinformasjonDto;
+import no.nav.sif.abac.kontrakt.abac.dto.SaksinformasjonTilgangskontrollInputDto;
+import no.nav.sif.abac.kontrakt.person.AktørId;
+import no.nav.sif.abac.kontrakt.person.PersonIdent;
+import no.nav.ung.sak.tilgangskontroll.api.AbacAttributter;
+
+import java.util.Arrays;
+
+public class PdpRequestMapper {
+
+    public static SaksinformasjonTilgangskontrollInputDto map(PdpRequest pdpRequest){
+        return new SaksinformasjonTilgangskontrollInputDto(
+            pdpRequest.getListOfString(AbacAttributter.RESOURCE_FELLES_PERSON_AKTOERID_RESOURCE).stream().map(AktørId::new).toList(),
+            pdpRequest.getListOfString(AbacAttributter.RESOURCE_FELLES_PERSON_FNR).stream().map(PersonIdent::new).toList(),
+            operasjon(pdpRequest),
+            saksinformasjon(pdpRequest)
+        );
+    }
+
+    public static OperasjonDto operasjon(PdpRequest pdpRequest) {
+        ResourceType resource = resourceTypeFraKode(pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.RESOURCE_FELLES_RESOURCE_TYPE));
+        return new OperasjonDto(resource, actionFraKode(pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.XACML_1_0_ACTION_ACTION_ID)));
+    }
+
+    public static SaksinformasjonDto saksinformasjon(PdpRequest pdpRequest) {
+        return new SaksinformasjonDto(
+            pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.RESOURCE_K9_SAK_ANSVARLIG_SAKSBEHANDLER),
```

Comment:
Burde det vært eget AbacAttributt for ung-sak: `RESOURCE_K9_SAK_ANSVARLIG_SAKSBEHANDLER`?